File: app/src/main/java/com/hqq/example/ui/dialog/DialogIndexActivity.java
package com.hqq.example.ui.dialog;

import com.hqq.core.ui.base.BaseListActivity;
import com.hqq.example.adapter.MainAdapter;
import com.hqq.example.bean.MainBean;

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.coreapp.ui.activity.dialog
 * @FileName :   DialogIndexActivity
 * @Date : 2019/5/25 0025  下午 2:35
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
public class DialogIndexActivity extends BaseListActivity<MainAdapter> {
    @Override
    public MainAdapter initAdapter() {
        return  new MainAdapter();
    }

    @Override
    public void initData() {
        mAdapter.addData(new MainBean("仿知乎评论列表", BottomSheetActivity.class));
        mAdapter.addData(new MainBean("dialog样式测试", TestDialogActivity.class));
    }


}

File: app/src/main/java/com/hqq/example/ui/dialog/DialogIndexActivity.kt
package com.hqq.example.ui.dialog

import com.hqq.core.ui.base.BaseListActivity
import com.hqq.example.adapter.MainAdapter
import com.hqq.example.bean.MainBean

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.coreapp.ui.activity.dialog
 * @FileName :   DialogIndexActivity
 * @Date : 2019/5/25 0025  下午 2:35
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
class DialogIndexActivity(override val baseAdapter: MainAdapter?=MainAdapter()) : BaseListActivity<MainAdapter?>() {

    override fun initData() {
        baseAdapter?.addData(MainBean("仿知乎评论列表", BottomSheetActivity::class.java))
        baseAdapter?.addData(MainBean("dialog样式测试", TestDialogActivity::class.java))
    }
}